includes/cli/class-co-authors-plus.php

```diff
@@ -15,11 +15,64 @@
  * Co-Authors Plus CLI commands.
  */
 class Co_Authors_Plus {
+	const NEWSPACK_SCHEDULE_AUTHOR_TERM_BACKFILL = 'newspack_cap_author_term_backfill';
+
 	private static $live = false; // phpcs:ignore Squiz.Commenting.VariableComment.Missing
 	private static $verbose = true; // phpcs:ignore Squiz.Commenting.VariableComment.Missing
 	private static $user_logins = false; // phpcs:ignore Squiz.Commenting.VariableComment.Missing
 	private static $guest_author_ids = false; // phpcs:ignore Squiz.Commenting.VariableComment.Missing
 
+	/**
+	 * Add hooks.
+	 */
+	public static function init() {
+		add_action( self::NEWSPACK_SCHEDULE_AUTHOR_TERM_BACKFILL, [ __CLASS__, 'run_cap_cli_command' ] );
+		add_action( 'init', [ __CLASS__, 'register_deactivation_hook' ] );
+	}
+
+	/**
+	 * Execute the co-authors-plus create-terms-for-posts CLI command.
+	 */
+	public static function run_cap_cli_command() {
+		if ( method_exists( 'WP_CLI', 'runcommand' ) ) {
+			$result = WP_CLI::runcommand(
+				'co-authors-plus create-author-terms-for-posts --batched --records-per-batch=50',
+				[
+					'launch' => false, // This keeps any formatting that's been set.
+					'return' => 'all', // This ensures we get stdout, stderr, and return code.
```

Comment:
By default, if `WP_CLI::runcommand()` encounters errors during script execution it will exit without reporting the results unless `'exit_error' => false` is set in the options array. I tested by manually adding a `WP_CLI::error()` command in the CAP plugin CLI command, and confirmed that adding this arg lets us log the error message.

```suggestion
					'exit_error' => false, // This allows us to capture any errors that occur during script execution.
					'launch'     => false, // This keeps any formatting that's been set.
					'return'     => 'all', // This ensures we get stdout, stderr, and return code.
```